src/OpenPostbud/workers/registration_worker.py

```diff
@@ -5,18 +5,19 @@
 
 import time
 from datetime import datetime
-import os
+import logging
 
-import dotenv
 from python_serviceplatformen import digital_post
 from python_serviceplatformen.authentication import KombitAccess
 from sqlalchemy import select, update
 
+from OpenPostbud import config
 from OpenPostbud.database import connection
 from OpenPostbud.database.check_registration.registration_task import RegistrationTask, TaskStatus
 from OpenPostbud.database.check_registration import registration_job
 
-dotenv.load_dotenv()
+logging.basicConfig(level=logging.INFO, format="%(levelname)s | %(name)s | %(asctime)s: %(message)s")
```

Comment:
Should logging format be moved into configuration and maybe use an std. logging format for parsing or sending logs into Loki?